backend/Origam.DA.Service/ReferenceIndexManager.cs

```diff
@@ -132,7 +150,14 @@ private static void GetReferencesFromText(AbstractSchemaItem item)
             }
         }
     }
-
+    private static void Addreference(AbstractSchemaItem retrievedObj)
```

Comment:
Why is the parameter called "retrievedObj". I think it would be better to call it "item" and the variable in the foreach below "child"